include/tvm/build_module.h

```diff
@@ -28,7 +29,7 @@ struct Target {
   /*! \brief The warp size that should be used by the LowerThreadAllreduce pass */
   int thread_warp_size = 1;
   /*! \brief Keys for this target */
-  std::unordered_set<std::string> keys;
+  std::vector<std::string> keys;
   /*! \brief Options for this target */
```

Comment:
I think it might make sense to also bring Target to the Node system, so current Target can simply return a None which can be checked with target.defined()